content/blog/vendasta.md

```diff
@@ -1,16 +1,15 @@
 ---
 title: Why we have decided to move our APIs to gRPC
-attribution: Originally written by Dale Hopkins with additional content by Lisa Carey
-  and others at Google.
-author: Dale Hopkins
-company: Vendasta
-company-link: https://vendasta.com
 date: 2016-08-29
-thumbnail: ../img/vend-icon.png?raw=true
+author:
+  name: Dale Hopkins
+  position: CTO, [Vendasta](https://vendasta.com)
+  guest: true
+  blurb: Originally written by Dale Hopkins with additional content by Lisa Carey and others at Google
+thumbnail: /img/vend-icon.png?raw=true
```

Comment:
Is the `?raw=true` really necessary?

Reply:
These thumbnails are not currently being used. I'm not quite sure what to do with them just yet so I'm leaving these as-is.